modules/salt/pages/salt-intro.adoc

```diff
@@ -2,19 +2,11 @@
 = Introduction
 
 
+Salt is a configuration management system used by {productname} to manage clients.
+Since {productname}{nbsp}3, you have the option of registering salt clients.
```

Comment:
```suggestion
You have the option of registering Salt clients.
```

Reply:
So I was trying to include the information about it being available in SUSE Manager 3+, but got  stuck on wording. Suggestions? Or maybe it's not important which version it was introduced  in?